tests/device_tests/test_protection_levels.py

```diff
@@ -261,23 +261,21 @@ def test_pin_cached(self, client):
         assert client.features.pin_cached is False
 
         with client:
-            client.set_expected_responses(
-                [proto.PinMatrixRequest(), proto.ButtonRequest(), proto.Address()]
-            )
-            btc.get_address(client, "Bitcoin", [], show_display=True)
+            client.set_expected_responses([proto.PinMatrixRequest(), proto.Address()])
+            btc.get_address(client, "Testnet", [0])
```

Comment:
Just wondering - why did you remove Testnet above but left it here?

Reply:
basically because it's a required argument for `get_address`, see commit message for https://github.com/trezor/trezor-firmware/pull/803/commits/7377c204d61af74d2f3051972c272513a1911b37

and I wanted to make the diff the smallest, so I used the parameters that were most common in the original changeset, which is `"Testnet", [0]` as opposed to `[]` which is also allowed